pkg/analyze/ceph.go

```diff
@@ -173,3 +227,18 @@ func compareCephStatus(actual, when string) (bool, error) {
 		return false, errors.New("unknown operator")
 	}
 }
+
+func detailedMessage(msg string, status CephStatus) string {
+	if status.OsdMap.OsdMap.Full {
+		msg = fmt.Sprintf("%s: OSD is full", msg)
+	} else if status.OsdMap.OsdMap.NearFull {
+		msg = fmt.Sprintf("%s: OSD is nearly full", msg)
+	}
+
+	if status.PgMap.TotalBytes > 0 {
+		pgUsage := 100 * float64(status.PgMap.UsedBytes) / float64(status.PgMap.TotalBytes)
+		msg = fmt.Sprintf("%s: PG usage is %.1f%%", msg, pgUsage)
```

Comment:
@jeffreygolden i basically went with what we discussed here but maybe we should change this to something a bit more human readable (e.g. "disk is nearly full: disk usage is 85%"). also lmk if you want some additional info in here.

Reply:
yeah i'm thinking more for custom messages but i'm not sure how often that feature is used